projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -630,17 +625,10 @@ public AnswerElement answerAclReachability(
       for (Entry<AclLine, Boolean> e : output.entrySet()) {
          AclLine line = e.getKey();
          String hostname = line.getHostname();
-         Map<String, List<AclLine>> byAclName = arrangedAclLines.get(hostname);
```

Comment:
I considered this pattern earlier and rejected it. I really don't remember why. Perhaps it was because I wanted to have a place to set breakpoints when a key was absent. In any case, that can be done ad hoc in the future if really necessary.